README.md

```diff
@@ -54,6 +54,7 @@ Collabora | https://www.collabora.com/ |
 Compose | https://www.compose.io/ |
 Consumer Financial Protection Bureau | http://www.consumerfinance.gov |
 CoreOS | https://www.coreos.com/ |
+Corgibytes | http://corgibytes.com | US East Coast
```

Comment:
Added only US East Coast as they have a preference for people in the same timezone.